app/models/manageiq/providers/embedded_terraform/automation_manager/job.rb

```diff
@@ -178,10 +178,7 @@ def input_vars_type_constraints
   end
 
   def input_vars
-    extra_vars = options.dig(:input_vars, :extra_vars) || {}
-    input_vars = options.dig(:input_vars, :input_vars) || {}
-    # merge & over extra_vars with input_vars
-    extra_vars.deep_merge!(input_vars)
+    ServiceTerraformTemplate.get_input_vars_from_job_options(options)
```

Comment:
Do we need to handle the old style in two different places?  Seems like the Service when ordered could format the input_vars properly so the Job can just take the new style

Reply:
:+1: awesome